### ISSUE TITLE
Need more releases

### ISSUE THREAD

**Issue by subsoap** [rgrams/rendercam#26]
Latest release version is old

**Comment by rgrams** [comment]
Yeah . . . sorry I have been absent the last few months. I'm sure you know there are a couple extra features on the 2.0 Beta branch, and some breaking changes on the Dev branch. Both should work just fine, I just haven't tested them to my satisfaction to call them released. You probably should use a local copy rather than a link to the Dev branch if you want that because when I get back to this I will break things there. 

I could add a note about this on the main readme, but unfortunately I probably won't do more actual work on it until late August. Feel free to fork it and do whatever you want of course.

**Comment by subsoap** [comment]
I understand. Do your best with what you've gotta do!

**Comment by rgrams** [comment]
...One year later...

I am finally starting to give Rendercam some love. I have a huge list of things I want to do, so I am slowly working towards a full v2.0 release. Check the dev branch if you are curious (but don't use it for any production projects).

**Comment by subsoap** [comment]
Awesome! Let me know when you need testing done. I'll make a new FPS project based on new version.
